src/audio-worklet/basic/audio-worklet-node-options/main.js

```diff
@@ -3,30 +3,39 @@
 // found in the LICENSE file.
 
 const audioContext = new AudioContext();
+let oscillatorProcessor;
 
 const startAudio = async (context, options) => {
-  await context.audioWorklet.addModule('oscillator-processor.js');
-  const oscillatorProcessor =
+  if (!oscillatorProcessor) {
+    await context.audioWorklet.addModule('oscillator-processor.js');
```

Comment:
This can't be called multiple times in any scenario. I think you'll get an error if you do this. Have you checked?

Reply:
Yes, it can't be called multiple times, so I have modified it. No, I didn't get any error though!